modules/data_release/ajax/GetFile.php

```diff
@@ -58,8 +58,12 @@
     header("HTTP/1.1 403 Forbidden");
     exit;
 }
-$fp = fopen($FullPath, 'r');
-fpassthru($fp);
-fclose($fp);
+
+// Output file in downloadable format
+header('Content-Description: File Transfer');
+header('Content-Type: application/force-download');
```

Comment:
`application/force-download` isn't a valid Mime type, and Content-Type is supposed to be the mime type of the content being served.

The `Content-Disposition` header below should be sufficient to force a download.